bench/wtperf/wtperf.c

```diff
@@ -1668,7 +1668,7 @@ execute_workload(CONFIG *cfg)
 	for (threads = cfg->workers, i = 0,
 	    workp = cfg->workload; i < cfg->workload_cnt; ++i, ++workp) {
 		lprintf(cfg, 0, 1,
-		    "Starting workload #%d: %" PRId64 " threads, inserts=%"
+		    "Starting workload #%u: %" PRId64 " threads, inserts=%"
```

Comment:
Actually, `cfg->workload_cnt` is a `u_int`, so I think this change is correct, and `i ` should stay a `u_int`. (Or they both should change?)